strongbox-client/src/main/java/org/carlspring/strongbox/client/ArtifactSpringClient.java

```diff
@@ -1,566 +0,0 @@
-package org.carlspring.strongbox.client;
```

Comment:
I'm not quite sure why these clients have been removed?
What am I missing?

Reply:
`ArtifactSpringClient` is not used in the codebase and similar/same methods are able to be found in other places in the codebase which are actually "in use".  Many times when I was dealing with ArtifactResolver I was checking what `ArtifactSpringClient` is for, what's the difference between them and the result was always the same. `ArtifactSpringClient` is currently useless. Would you like me to roll it back ?